AnimatedEvent.web.js

```diff
@@ -0,0 +1,3 @@
+import AnimatedEvent from 'react-native-web/dist/vendor/react-native/Animated/AnimatedEvent';
```

Comment:
If `AnimatedEvent` needs to be used directly, you may want to ask that it is added to the `Animated` export from React Native rather than relying on the fragility of reaching into internals.

Reply:
Okay, I will check on their repository and raise a PR.

Reply:
`AnimatedEvent` is just used as a type. This shouldn't prohibit any functionality.